### DISCUSSION TITLE
Latest Emgu.CV nuget install to VS Community 2019 fail

### DISCUSSION THREAD

**Discussion by MagnusGabell** [emgucv/emgucv#401] [General]
Hi,

Just want to make you aware that the latest version 4.5 does not install to VSC 2019.
The nuget package installar for runtime complains about missing assembly references.
I tried to install the dependencies also seperately but it failed too.

Rolling back to 4.4 works!

**Comment by nombrote** [comment]
Yes, it fail with .Net Framework projects, dont have the reference. Only work on .Net core. I spect it's repared eary.

I have the some problem.

**Reply by jleoutward** [reply]
I notice the same thing too.  Couldn't get Emgu.CV.runtime.windows.4.5.1.4349 to install on VS2017 with .Net Framework 4.8.

Here's another discussion with some screenshot of what I see:
https://github.com/emgucv/emgucv/discussions/398#discussioncomment-315753

**Comment by emgucv** [comment]
It works with ".Net Framework" too, just need to make sure that you have upgraded from "packages.config" to "Package reference". 

Nuget dependency failed to resolve when using "packages.config" for nuget packages. Upgrading that to "Package reference" fixed that.

Detail can be found in this issue here: https://github.com/emgucv/emgucv/issues/399#issuecomment-767571013

**Reply by MagnusGabell** [reply]
Thank you! I will test that and respond with my success!
